libs/domain/cart/list-item/stories/Static/variants.stories.ts

```diff
@@ -0,0 +1,37 @@
+import { Meta, Story } from '@storybook/web-components';
+import { TemplateResult, html } from 'lit';
+import { storybookPrefix } from '../../../.constants';
+
+export default {
+  title: `${storybookPrefix}/List Item/Static`,
```

Comment:
```suggestion
  title: `${storybookPrefix}/List Item`,
```

Reply:
But this is a static story, we are keeping them in the separate sub-folder in storybook. Or do we have a new rule about this?